src/tirith/providers/common.py

```diff
@@ -5,6 +5,11 @@ def create_result_dict(value=None, meta=None, err=None) -> Dict:
     return dict(value=value, meta=meta, err=err)
 
 
+def get_path_value_from_dict(key_path, input_dict,get_path_value_from_dict_func):
```

Comment:
Missing one space on the third param (sorry for the nitpicky :))
I'd also add parameters types while making this change, e.g. `key_path: str`

Reply:
Heyy, sure. I'll make these changes.

Reply:
Heyy @refeed , Have made the changes :)